packages/x-license-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "6.0.3",
+  "version": "6.0.4",
```

Comment:
For this package, I'm honestly not sure... Technically, there is nothing warranting a minor release, so, maybe sticking to the usual semver makes the most sense. 🤔